src/Passes/Source/AllocationManager/AllocationManager.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "AllocationManager/AllocationManager.hpp"
```

Comment:
This is a good suggestion. I will not make any comments on this at this point in time as I do not really know how the runtime allocator works. I think it is still very early days for the compiler allocations so it is hard to estimate what exactly is needed here in the long run. It may be that we can reuse parts of the runtime, or might make sense to make a common interface. 

I will leave this for now as I think this is a wider discussion to have with others involved as well.